src/app/models/question.rb

```diff
@@ -1,5 +1,5 @@
 class Question < ApplicationRecord
-	belongs_to :quiz
-	has_many :answers, :dependent => :destroy
-	accepts_nested_attributes_for :answers, :reject_if => lambda { |a| a[:content].blank? }, :allow_destroy => true
+	has_many :answers
+	belongs_to :quizzes
+	accepts_nested_attributes_for :answers, allow_destroy: true
```

Comment:
Why were these changes made? The lambda prevented blank questions from existing, and the `dependent => destroy` said that if the question is deleted, delete its answers as well.